### PR TITLE
Update wordpress to schemaV2 and remove wordpress-updatable

### PR DESCRIPTION
**Pull request by eshiroma**
And now-unused GCS publish functionality.

/gcbrun

### CODE REVIEW

File: wordpress/Makefile [hunk 1]
```diff
@@ -39,23 +51,25 @@ app/build:: .build/wordpress/deployer \
...
-                           deployer/* \
-                           manifest/* \
-                           schema.yaml \
-                           | .build/wordpress
+                                     .build/var/MARKETPLACE_TOOLS_TAG \
+                                     .build/var/REGISTRY \
+                                     .build/var/TRACK \
+                                     .build/var/RELEASE \
+                                     $(shell find apptest/deployer -type f) \
+                                     $(shell find apptest/deployer/manifest -type f) \
```
Comment (gibbleyg) [inline wordpress/Makefile:59]:
```suggestion
                                     $(shell find apptest/deployer -type f) \
```

Or we can do a single
$(shell find . -mindepth 2 -type f)

**Review by gibbleyg — APPROVED** [review]

File: wordpress/Makefile (hunk 1 above)
Comment (eshiroma) [reply wordpress/Makefile]: Done.

Comment (eshiroma) [reply wordpress/Makefile]: Decided to keep `shell find`s for readability, but removed the redundant `$(shell find apptest/deployer/manifest -type f)`

**Pull request merged**